include/nbl/builtin/hlsl/spirv_intrinsics/core.hlsl

```diff
@@ -189,10 +189,19 @@ template<typename T, typename P>
 enable_if_t<is_spirv_type_v<P>,void> store(P pointer, T obj);
 
 //! Std 450 Extended set operations
+
+// COMMENTING OUT BECAUSE OF https://github.com/microsoft/DirectXShaderCompiler/issues/6751, bring back when fixed!
+/*
+
 template<typename SquareMatrix>
 [[vk::ext_instruction(GLSLstd450MatrixInverse)]]
 SquareMatrix matrixInverse(NBL_CONST_REF_ARG(SquareMatrix) mat);
 
+[[vk::ext_instruction(GLSLstd450UnpackSnorm2x16)]]
+float32_t2 unpackSnorm2x16(uint32_t p);
+
+*/
```

Comment:
undo this and add the set https://github.com/microsoft/DirectXShaderCompiler/issues/6751

Reply:
resolved